.env.test

```
@@ -0,0 +1,10 @@
+# Flask server configuration.
+FLASK_APP=todo_app/app
+FLASK_ENV=development
+
+# Change the following values for local development.
+SECRET_KEY=secret-key
+
+TRELLO_API_KEY=123456789abcdef0123456789abcdef0
+TRELLO_TOKEN=123456789abcdef0123456789abcdef0123456789abcdef0123456789abcdef0123456789abc
```

I can see these are fake credentials, but I do have to look at them for a few seconds. I'm personally a fan of really blatantly obvious fake credentials to save people having to consider whether there's a risk here, e.g.:
```suggestion
TRELLO_API_KEY=fake-key
TRELLO_TOKEN=fake-token
```

---

Interesting.  I would want to use valid values that express the range of valid characters and going to the expected length.